docs/content/posts/2024-10-01-release-2.0-preview.md

```diff
@@ -0,0 +1,99 @@
+
+## Misc
+
+**State Compatibility** is not guaranteed between 1.x and 2.x. 
+
+**Java 8** is no longer supported. The minimum Java version supported by Flink now is Java 11.
+
+**Per-Job Deployment Mode** is removed. Please use [Application Mode](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/overview/#application-mode) instead.
+
+**Legacy Mode of Hybrid Shuffle** is removed.
+
+# Highlights of New Features
+
+## Disaggregated State Storage and Management
+
+The past decade has witnessed a dramatic shift in Flink's deployment mode, workload patterns, and hardware improvements. We've moved from the map-reduce era where workers are computation-storage tightly coupled nodes to a cloud-native world where containerized deployments on Kubernetes become standard. To enable Flink's Cloud-Native future, we introduce Disaggregated State Storage and Management that uses Distributed File Systems (DFS) as primary storage in Flink 2.0.
+
+This new architecture solves the following challenges brought in the cloud-native era for Flink. 
+1. Local Disk Constraints in containerization 
+2. Spiky Resource Usage caused by compaction in the current state model 
+3. Fast Rescaling for jobs with large states (hundreds of Terabytes) 
+4. Light and Fast Checkpoint in a native way 
+
+However, simply extending the state store to read/write from remote DFS is insufficient due to the existing blocking execution model in Flink. In Flink 2.0, we propose asynchronous execution model and introduce ForStDB, a disaggregated statebackend solution for this purpose.
+
+In the preview version, we offer a complete end-to-end trial using Nexmark Q20 (SQL Filter Join). This includes: 
+- Asynchronous execution: Full support for asynchronous state APIs and checkpointing. 
+- Asynchronous SQL Join operator: Rewrite SQL Join operators to enable asynchronous join execution.
+- Hybrid Async & Sync Execution: Hybrid SQL Plan + Runtime Execution + State Access
+- Performance: Demonstrate performance results directly writing to DFS in async execution mode
+
+## Materialized Table
+
+In Flink 1.20, we introduced Materialized Table as a MVP feature. Materialized Table is an innovative table type in Flink SQL, designed to further streamlining batch and stream data processing while providing a unified developement experience. In the upcoming Flink 2.0 release, we are enhancing operational supports for Materialized Tables, including connector integration with cutting-edge lake formats and production-ready schedulers.
+
+## Adaptive Batch Execution
+
+In addition, Flink is countinuously enhancing its adaptive batch execution capabilities. The upcoming Flink 2.0 release will introduce dynamic optimization of logical plans, in addition to physical plans, based on insights gained from the execution of previous stages. The initial set of optimization strategies includes the dynamic application of broadcast joins and skewed joins.
```

Comment:
> dynamic application of ... skewed joins

Looks to me it is not proper because `broadcast join` is the goal in that case, while `skewed join` is the problem to solve.